benchmarks/benches/timeline.rs

```diff
@@ -118,18 +118,7 @@ pub fn create_timeline_with_initial_events(c: &mut Criterion) {
 }
 
 fn criterion() -> Criterion {
-    #[cfg(target_os = "linux")]
-    {
-        Criterion::default().with_profiler(pprof::criterion::PProfProfiler::new(
-            100,
-            pprof::criterion::Output::Flamegraph(None),
-        ))
-    }
-
-    #[cfg(not(target_os = "linux"))]
-    {
-        Criterion::default()
-    }
+    Criterion::default()
```

Comment:
Nice. Any chance we can remove the `criterion()` function altogether, and remove the `config = criterion()` in the `criterion_group!` macro invoke? (If so, we could do this for every file)

Reply:
I intended to leave this in as it will be easier to bring back local pprof support, but sure we can remove it if you prefer the functions to be gone.